consensus/dummy/dummy.go

```diff
@@ -0,0 +1,126 @@
+package dummy
+
+import (
+	"context"
+	"time"
+
+	"github.com/0xPolygon/minimal/blockchain"
+	"github.com/0xPolygon/minimal/consensus"
+	"github.com/0xPolygon/minimal/network"
+	"github.com/0xPolygon/minimal/state"
+	"github.com/0xPolygon/minimal/txpool"
+	"github.com/0xPolygon/minimal/types"
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc"
+)
+
+type Dummy struct {
+	sealing    bool
+	logger     hclog.Logger
+	notifyCh   chan struct{}
+	closeCh    chan struct{}
+	txpool     *txpool.TxPool
+	blockchain *blockchain.Blockchain
+	executor   *state.Executor
+}
+
+func Factory(ctx context.Context, sealing bool, config *consensus.Config, txpool *txpool.TxPool, network *network.Server, blockchain *blockchain.Blockchain, executor *state.Executor, srv *grpc.Server, logger hclog.Logger) (consensus.Consensus, error) {
+	logger = logger.Named("dummy")
+
+	d := &Dummy{
+		sealing:    sealing,
+		logger:     logger,
+		notifyCh:   make(chan struct{}),
+		closeCh:    make(chan struct{}),
+		blockchain: blockchain,
+		executor:   executor,
+		txpool:     txpool,
+	}
+
+	txpool.NotifyCh = d.notifyCh
+
+	return d, nil
+}
+
+func (d *Dummy) Start() error {
+	go d.run()
+	return nil
+}
+
+func (d *Dummy) VerifyHeader(parent *types.Header, header *types.Header) error {
+	// All blocks are valid
+	return nil
+}
+
+func (d *Dummy) Close() error {
+	close(d.closeCh)
+	return nil
+}
+
+func (d *Dummy) run() {
```

Comment:
Is there any point in sealing blocks, the goal of the dummy consensus is to enable everything on the network side of things (txpool and discovery) but without the hassle of syncing logic. I think the run function can be removed.